sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/tests/Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests/DistroWebAppLiveTests.cs

```diff
@@ -0,0 +1,139 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+using Azure.Core.TestFramework;
+using Azure.Monitor.Query;
+using Azure.Monitor.Query.Models;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using NUnit.Framework;
+using OpenTelemetry.Logs;
+using OpenTelemetry.Resources;
+
+#if NET6_0_OR_GREATER
+namespace Azure.Monitor.OpenTelemetry.AspNetCore.Integration.Tests
+{
+    public class DistroWebAppLiveTests : RecordedTestBase<MonitorExporterTestEnvironment>
```

Comment:
Do you want to use this name `MonitorExporterTestEnvironment`?

Reply:
That class already exists, but was migrated from the Exporter Integration Tests. I can rename it

Reply:
You could cover in a follow up PR.